azurerm/helpers/validate/web_application_firewall_policy.go

```diff
@@ -28,7 +28,7 @@ var ValidateWebApplicationFirewallPolicyRuleGroupName = validation.StringInSlice
 }, false)
 
 var ValidateWebApplicationFirewallPolicyRuleSetVersion = validation.StringInSlice([]string{
-	"1.0",
+	"0.1",
```

Comment:
generally speaking when Azure deprecates something, the existing value will continue working for existing deployments - but it can't be used for new deployments; as such we need to add `1.0` back in here:

```suggestion
	"0.1",
	"1.0",
```